### PR TITLE
Merge init rails app to master

### PR DESCRIPTION
**Pull request by Zawgor24**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
```
Comment (IlyaUmanets) [inline app/assets/javascripts/application.js]: Remove commented lines. Keep only `require` lines

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# PostgreSQL. Versions 9.1 and up are supported.
```
Comment (IlyaUmanets) [inline config/database.yml]: You should to ignore this file. Create `database.example.yml` file instead.

File: config/environments/development.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
```
Comment (IlyaUmanets) [inline config/environments/development.rb]: Remove commented lines

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]
Remove all commented lines. Try to review your PR by yourself.

File: config/database.yml (hunk 2 above)
Comment (Zawgor24) [reply config/database.yml]: Commented lines were removed, and added new database.example file.

**Pull request merged**